torque_bias:
- -0.0 # joint 1
- -0.0 # joint 2
- -0.0 # joint 3
- 0.2 # joint 4
- 0.0 # joint 5
- 0.0 # joint 6
- 0.0 # joint 7
torque_stiction:
- 0.45 # joint 1
- 0.55 # joint 2
- 0.35 # joint 3
- 0.38 # joint 4
- 0.28 # joint 5
- 0.25 # joint 6
- 0.15 # joint 7
panda_address: "panda"
netft_address: "192.168.11.3"
withNetft: True
withGripper: True
